pkg/defaults/defaults.go

```diff
@@ -307,7 +309,26 @@ func FromConfig(
 		if err != nil {
 			return nil, nil, fmt.Errorf("could not determine promotion defaults: %w", err)
 		}
-		postSteps = append(postSteps, release.PromotionStep(*cfg, config.Images, requiredNames, imageClient, imageClient, jobSpec))
+
+		dstImageClient := imageClient
+
+		if promoteKubeConfigPath != "" {
+			kubeConfigs, _, err := util.LoadKubeConfigs(promoteKubeConfigPath)
+			if err != nil {
+				return nil, nil, fmt.Errorf("failed to load kubeconfigs: %w", err)
```

Comment:
Please clarify which kubeconfig this was and from where we attempted to load it

Reply:
Updated.